dusty/command_file.py

```diff
@@ -0,0 +1,129 @@
+import os
+from nose.tools import nottest
```

Comment:
Here, you can call `nottest` on the function after you've imported it in your test modules and then Nose will ignore it. https://github.com/gamechanger/dusty/blob/74f7c670d6623b39f620279e3f826a5174cee5b1/tests/unit/systems/docker/testing_images_test.py#L13